lib/srv/db/postgres/engine.go

```diff
@@ -202,7 +202,7 @@ func (e *Engine) handleStartup(client *pgproto3.Backend, sessionCtx *common.Sess
 	}
 	switch m := startupMessageI.(type) {
 	case *pgproto3.StartupMessage:
-		e.Log.Debugf("Received startup message: %#v.", m)
+		e.Log.DebugContext(e.Context, "Received startup message.", "message", fmt.Sprintf("%#v", m))
```

Comment:
We don't have a LogValuer for `%#v` (that I'm aware of), although I suppose we could make one.